src/SDKs/ServiceBus/ServiceBus.Tests/ServiceBus.Tests.csproj

```diff
@@ -10,6 +10,10 @@
   <PropertyGroup>
     <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
   </PropertyGroup>
+
```

Comment:
@v-Ajnava you do not need this if you take the latest from upstream and execute msbuild build,proj